tests/test_client.py

```diff
@@ -413,7 +455,7 @@ def create_batch_post(self, stream, num_records):
     ### UPDATEs
     ##########################################################################
 
-    def update(self, stream, obj_id, version):
+    def update(self, stream, obj_id, version, obj=None):
```

Comment:
As I said earlier, can we change `obj` to something slightly less generic like `stream_data` or `stream_sample_record`?